packages/ui-lite/src/EventBus.ts

```diff
@@ -53,8 +53,8 @@ class EventBus extends EventEmitter2 {
     socket && socket.emit('event', { name, data })
   }
 
-  updateVisitorId = (newId: string, userIdScope?: string) => {
-    setVisitorId(newId, userIdScope)
+  deleteVisitorId = (userIdScope?: string) => {
+    unsetVisitorId(userIdScope)
```

Comment:
you dont use unset anywhere, not sure why you did not just put the logic here instead of adding a new method

Reply:
Unset is called by `deleteVisitorId` and `deleteVisitorId` is called by `newUserId` in `modules/channel-web/src/views/lite/core/socket.tsx` and `newUserId` is called when a new session is requested